lib/pure/unittest.nim

```diff
@@ -18,12 +18,16 @@
 ## overridden locally.
 ##
 ## Compiled test files return the number of failed test as exit code, while
-## ``nim c -r <testfile.nim>`` exits with 0 or 1
 ##
-## Running a single test
-## =====================
+## .. code::
+##   nim c -r testfile.nim
+##
+## exits with 0 or 1.
+##
+## Running individual tests
+## ========================
 ##
-## Specify the test name as a command line argument.
+## Specify the test names as command line arguments.
```

Comment:
Thank you for rewording, it sounds better. I really wish more people appreciated polishing these kinds of things :)